### PR TITLE
Few improvements to the demo app

### PR DESCRIPTION
**Pull request by ravimandala**

### CODE REVIEW

File: sample/build.gradle [hunk 1]
```diff
@@ -50,6 +50,7 @@ android {
             signingConfig signingConfigs.release
 		}
 	}
+    dataBinding.enabled = true
```
Comment (annavungle) [inline sample/build.gradle]: what is the purpose of dataBinding? We don't have that setting specified in the current Integration guide

File: sample/proguard-project.txt [hunk 2]
```diff
@@ -16,5 +16,6 @@
 #   public *;
 #}
 -dontwarn com.vungle.**
+-dontnote com.vungle.**
```
Comment (annavungle) [inline sample/proguard-project.txt]: Should we add this proguard setting in our guide as well?

File: sample/build.gradle (hunk 1 above)
Comment (annavungle) [inline sample/build.gradle]: "Data Binding Library to write declarative layouts and minimize the glue code necessary to bind your application logic and layouts." from Android doc

**Comment by ravimandala** [comment]
Yes, that is for the Sample App's UI. Nothing to do with SDK or the
integration.

**Pull request closed without merging**